apps/server-asset-sg/src/main.ts

```diff
@@ -12,6 +12,7 @@ const API_PORT = process.env.PORT || 3333;
 async function bootstrap(): Promise<void> {
   const app = await NestFactory.create(AppModule);
   app.setGlobalPrefix(API_PREFIX);
+  app.useGlobalPipes(new ValidationPipe({ transform: true, whitelist: true, forbidNonWhitelisted: true }));
```

Comment:
Does this mean that validation was inactive before?

Reply:
No, it was handled locally, i.e. every route that wanted to use validation had to specify it themselves. Now that we have fully rewritten all API paths, we can enable it globally.